include/Track.h

```diff
@@ -165,6 +186,8 @@ public slots:
 	void lengthChanged();
 	void positionChanged();
 	void destroyedTCO();
+	void trackColorChanged();
+	void trackColorReset();
```

Comment:
The `trackColorReset` signal is never emitted - remove it?

Reply:
It was linked to a whole bunch of functions, but those functions were made redundant by changing how colors were displayed, so I removed the links too.

Reply:
Also tracked down one more function - `trackColorReset()` (a different function in another class) - which has been made redundant, got rid of it as well.